src/famfs_lib.c

```diff
@@ -707,6 +720,13 @@ famfs_mmap_superblock_and_log_raw(const char *devname,
 	int openmode = (read_only) ? O_RDONLY : O_RDWR;
 	int mapmode  = (read_only) ? PROT_READ : PROT_READ | PROT_WRITE;
 
+	assert(sbp); /* superblock pointer mandatory; logp optional */
+
+	if (log_size &&
+	    ((log_size < FAMFS_LOG_LEN) || (log_size & (0x200000 - 1)) )) {
```

Comment:
Shouldn't this (log_size & (0x200000 - 1))   be   (log_size & (log_size - 1))  assuming we are testing if it is a power of 2 value  ?

(0x200000 - 1) = 1FFFFE 
logsize & 1FFFFE seems not correct to me. 
I wonder how the tests are passing... am i missing something...

Reply:
Yes, thanks